.circleci/config.yml

```diff
@@ -346,6 +365,72 @@ jobs:
       - store_artifacts:
           path: ~/junit
 
+  analyze-pull-request:
+    <<: *defaults
+    docker:
+      - image: circleci/node:8
+    steps:
+      - checkout
+      - restore-cache: *restore-cache-analysis
+      - run:
+          name: Install Dependencies
+          command: |
+            if [ -n "$CIRCLE_PULL_REQUEST" ]; then
+              npm config set spin=false
+              npm config set progress=false
+              npm install --no-package-lock
+              npm install github@0.2.4
+              cd danger
+              npm install --no-package-lock
+            else
+              echo "Skipping dependency installation."            
+            fi
+      - save-cache: *save-cache-analysis
+# Run Danger      
+      - run:
+          name: Analyze Pull Request
+          command: |
+            if [ -n "$CIRCLE_PULL_REQUEST" ]; then
+              cd danger && DANGER_GITHUB_API_TOKEN="e622517d9f1136ea8900""07c6373666312cdfaa69" npm run danger
+            else
+              echo "Skipping pull request analysis."            
+            fi
+          when: always
```

Comment:
Wish there was a way to run on pull request builds only in a more declarative way

Reply:
Same.